core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -612,6 +612,21 @@ private function addCrawlNode(): NodeDefinition
             ->getRootNode()
             ->addDefaultsIfNotSet()
             ->children()
+                ->integerNode('concurrency')
```

Comment:
Hmmm...should we nest this in `backend` because it only affects the backend UI and it has not influence on the `contao:crawl` command. Whereas `additional_uris` does.

Reply:
`contao.backend.crawl.concurrency` or `contao.backend.crawl_concurrency`?

Reply:
The first one, what do you think? Also I noticed that the defaults on CLI for concurrency is 10, do you think we should syncrhonize that between backend and CLI?

Reply:
No, I don‘t think we should. The web process is typically more limited than a CLI process.

Reply:
Why? This influences the number of concurrent requests. They are always served by the web? 😄

Reply:
Namespace changed in cbf4062f9a35c8e3addb227b5e5aa5c29d511e32.